src/components/cards/verticalCard/VerticalCard.jsx

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import "./verticalCard.css";
+
+function VerticalCard() {
+  return (
+    <div>
+      <div class="videolib-verticalcard-resize">
```

Comment:
Should be `className` instead. Fix it in all other components also.

Reply:
ohh I forget that , I will fix that

Reply:
Done